.husky/pre-commit

```diff
@@ -1,4 +1,4 @@
 #!/bin/sh
 . "$(dirname "$0")/_/husky.sh"
 
-npx lint-staged
+npx --no lint-staged
```

Comment:
[note] Use the local dependency. See <https://docs.npmjs.com/cli/v7/commands/npm-exec>.